test/extended/cli/set.go

```diff
@@ -0,0 +1,79 @@
+package cli
+
+import (
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	appsv1 "github.com/openshift/api/apps/v1"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+const deploymentRunTimeout = 5 * time.Minute
+const deploymentChangeTimeout = 30 * time.Second
+
+var _ = g.Describe("[cli][Slow] oc set", func() {
+	defer g.GinkgoRecover()
+
+	var (
+		oc                      = exutil.NewCLI("oc-set", exutil.KubeConfigPath())
+		simpleDeploymentFixture = exutil.FixturePath("testdata", "deployments", "deployment-simple.yaml")
+		imageSourceLabel        = exutil.ParseLabelsOrDie("name=deployment-simple")
+	)
+
+	g.Describe("volume", func() {
+		dcName := "deployment-simple"
+
+		g.It("should works well with DC", func() {
+			namespace := oc.Namespace()
+			dc := exutil.ReadFixtureOrFail(simpleDeploymentFixture).(*appsv1.DeploymentConfig)
+			o.Expect(dc.Name).To(o.Equal(dcName))
```

Comment:
This 2 lines are covered in test/extended/deployments/deployments.go. I think they are redundant and thus could be removed.